builtin/logical/transit/policy.go

```diff
@@ -25,6 +30,11 @@ const (
 	ErrTooOld = "ciphertext version is disallowed by policy (too old)"
 )
 
+const (
```

Comment:
Can we merge this and the above const blocks and remove `kdfMode`? (It and its comment is misleading)